src/components/Profile.tsx

```diff
@@ -1,19 +1,24 @@
 import React, { useState } from "react";
 import UserAvatar from "./UserAvatar";
-import { Button } from "antd";
 import ConnectionsList from "./ConnectionsList";
 import { useAppSelector } from "../redux/hooks";
 import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 import SettingsIcon from "../images/SettingsIcon.svg";
 import ArrowIcon from "../images/ArrowIcon.svg";
+import * as types from "../utilities/types";
 
 const Profile = (): JSX.Element => {
   const [showProfile, setShowProfile] = useState<boolean>(false);
   const userId: DSNPUserId | undefined = useAppSelector(
     (state) => state.user.id
   );
-  const handle = "Hans";
-  const profileName = "lovetoeat";
+
+  const profiles: Record<DSNPUserId, types.Profile> = useAppSelector(
+    (state) => state.profiles?.profiles || {}
+  );
+
+  const handle = userId && profiles[userId]?.handle;
```

Comment:
The `userId &&` here and on the next line is uneccessary. You can pass undefined as the key to an object (which will result in undefined).